README.md

```diff
@@ -1,117 +1,84 @@
 
-[![Build Status](https://jenkinsci.upbound.io/buildStatus/icon?job=crossplane/build/master)](https://jenkinsci.upbound.io/blue/organizations/jenkins/crossplane%2Fbuild/activity)
-[![GitHub release](https://img.shields.io/github/release/crossplaneio/crossplane/all.svg?style=flat-square)](https://github.com/crossplaneio/crossplane/releases)
-[![Docker Pulls](https://img.shields.io/docker/pulls/crossplane/crossplane.svg)](https://img.shields.io/docker/pulls/crossplane/crossplane.svg)
-[![Go Report Card](https://goreportcard.com/badge/github.com/crossplaneio/crossplane)](https://goreportcard.com/report/github.com/crossplaneio/crossplane)
-[![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Fcrossplaneio%2Fcrossplane.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2Fcrossplaneio%2Fcrossplane?ref=badge_shield)
-[![Slack](https://slack.crossplane.io/badge.svg)](https://slack.crossplane.io)
-[![Twitter Follow](https://img.shields.io/twitter/follow/crossplane_io.svg?style=social&label=Follow)](https://twitter.com/intent/follow?screen_name=crossplane_io&user_id=788180534543339520)
+![Crossplane](docs/media/banner.png)
 
-[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=crossplaneio_crossplane&metric=alert_status)](https://sonarcloud.io/dashboard?id=crossplaneio_crossplane)
-[![Coverage](https://sonarcloud.io/api/project_badges/measure?project=crossplaneio_crossplane&metric=coverage)](https://sonarcloud.io/dashboard?id=crossplaneio_crossplane)
-[![Maintainability Rating](https://sonarcloud.io/api/project_badges/measure?project=crossplaneio_crossplane&metric=sqale_rating)](https://sonarcloud.io/dashboard?id=crossplaneio_crossplane)
-[![Reliability Rating](https://sonarcloud.io/api/project_badges/measure?project=crossplaneio_crossplane&metric=reliability_rating)](https://sonarcloud.io/dashboard?id=crossplaneio_crossplane)
-[![Security Rating](https://sonarcloud.io/api/project_badges/measure?project=crossplaneio_crossplane&metric=security_rating)](https://sonarcloud.io/dashboard?id=crossplaneio_crossplane)
+[Crossplane] is the open source multicloud control plane. With Crossplane you
+can manage your applications and infrastructure across clouds, regions, and
+clusters. Crossplane is composable and Kubernetes native. Deploy it to an
+existing Kubernetes cluster to manage cloud infrastructure like databases,
+buckets, and caches using `kubectl`. Deploy it standalone to manage Kubernetes
```

Comment:
> it sounds like it's making the point that the only way to manage k8s clusters and workloads is if you deploy it standalone.

Hrm, my intent here is to highlight the two most common ways you might deploy and use Crossplane. I'm personally having trouble reading it back and taking away that it's saying you can _only_ manage Kubernetes clusters and workloads if you deploy Crossplane standalone.

> You could just as easily deploy it to an existing k8s cluster and still manage other k8s clusters and schedule workloads to them, right?

You _could_, but do you think this is a use case anyone has? I already run my own Kubernetes cluster(s), and I'm installing Crossplane there to manage other Kubernetes clusters and the workloads running on them? My feeling is that if this use case exists at all, it's a pretty rare one.

Reply:
My broader thinking here is:

* Crossplane is useful as a tool dropped into an existing Kubernetes cluster. My intuition is that this will be the most common on-ramp into Crossplane; folks who have a cluster and want to manage their infrastructure from inside Kubernetes.
* Crossplane's Kubernetes cluster and workload management functionality doesn't, in my opinion, make much sense in the context of a service installed into an existing Kubernetes cluster.  I think wanting to install a tool into your existing cluster to manage other clusters is a niche use case at best.
* The cluster and workload management functionality makes much more sense if you start to look at Crossplane as your multicloud control plane above the cluster level, but I'm not sure folks will make that "mental leap" without a little prompting. I certainly would not have.

So the goal here is to lead with "Hey, are you using Kubernetes today? If you drop Crossplane into your existing setup you'll get this value" but to also set context as to where Crossplane is going with all this cluster and workload management functionality.